### DISCUSSION TITLE
Is the Redis instrumentation library maintained?

### DISCUSSION THREAD

**Discussion by ghost** [open-telemetry/opentelemetry-dotnet#3414] [Q&A] [answered]
Hello,

I am noticing the redis instrumentation library is not on main, yet it's released on nuget.
I sit still maintained officially?

**Comment by cijothomas** [comment, marked as answer]
Redis instrumentation has been moved to contrib repo : https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md#100-rc95-source-code-moved-to-contrib-repo
